app/javascript/lib/sentry-logging.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @fileoverview Functions for logging JS errors to Sentry
+ *
+ */
+import * as Sentry from '@sentry/react'
+import getSCPContext from '~/providers/SCPContextProvider'
+
+/**
+ * Log an exception to Sentry for bad response JS fetch executions
+ * i.e. executions that results in a 404 response
```

Comment:
Blocker: with i.e. meaning "that is", this currently implies _only_ 404's are logged, which is incorrect.  

I'd go with e.g. ("for example"), but I defer to you as long as it's not i.e.

```suggestion
 * e.g. requests that result in a 404 response
```